Source/Components/Touchpoints/Types/Carousel/MLBusinessCarouselContainerView .swift

```diff
@@ -98,12 +99,21 @@ public class MLBusinessCarouselContainerView: UIView {
         ])
     }
     
+    private func setupCardSpace() {
+        for item in items {
+            if item.type?.uppercased() == Configuration.CardType.full {
+                layout.minimumLineSpacing = Configuration.Layout.minimumLineSpacing
+            }
+        }
```

Comment:
Pregunta, esto no sería mejor así?
```suggestion
        if items.contains { $0.type?.uppercased() == Configuration.CardType.full } {
            layout.minimumLineSpacing = Configuration.Layout.minimumLineSpacing
        }
```